share/doc/homebrew/Analytics.md

```diff
@@ -0,0 +1,41 @@
+# Analytics
+Homebrew will shortly begin gathering anonymous aggregate user behaviour analytics and reporting these to Google Analytics.
+
+## Why?
+Homebrew is provided free of charge and run entirely by volunteers in their spare time. As a result, we do not have the resources to do detailed user studies of Homebrew users to decide on how best to design future features and prioritise current work. Anonymous aggregate user analytics allow us to prioritise fixes and features based on how, where and when people use Homebrew.
```

Comment:
I think a clear and detailed example will help. Identifying buggy formulae is one, since we will report build failures.